vendor/cran.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
Maybe `services/tests/cran.js`? Generally tests should be in a `tests` directory (or `__tests__` like we use at Facebook), and `vendor` is generally reserved for third-party code.